models/README.md

```diff
@@ -1,24 +1,23 @@
-# Cortex Certifai Models  
+# Cortex Certifai Models
 
-The *models* folder contains examples of using the Certifai Model SDK to  
-wrap trained models in a prediction service  
-that can be scanned using the Certifai scanner CLI. See the *notebooks* folder  
-for examples of scanning models in a Jupyter notebook using the Certifai API.  
+The *models* folder contains examples of using the Certifai Model SDK to wrap trained models in a prediction service
+that can be scanned using the Certifai scanner CLI. See the *notebooks* folder for examples of scanning models in a
+Jupyter notebook using the Certifai API.
 
 
 ## Certifai Documentation
 
-Refer to the
-[Cortex Certifai documentation](https://cognitivescale.github.io/cortex-certifai/docs/about)
+Refer to the [Cortex Certifai documentation](https://cognitivescale.github.io/cortex-certifai/docs/about)
 for detailed information about Cortex Certifai.
 
 ## Models Table of Contents
 
-| Model Folder | Description | Task Type | Language | Model Framework |
-| --- | --- | --- | --- | -- |
```

Comment:
The reason this table did not render correctly is that the third dash is missing in the last column. You shouldn't need all those dashes.

Reply:
All the dashes are the result of Vim & Pycharm Plugin(s) I use that auto-format the table. It is a bit excessive with the dashes, but its just to line the all the cells up